activemodel/lib/active_model/type/helpers/time_value.rb

```diff
@@ -38,6 +42,11 @@ def user_input_in_time_zone(value)
 
         private
 
+          def value_is_rounding_candidate?(value)
+            value.respond_to?(:usec) &&
+              (value.usec.positive? || value.respond_to?(:nsec) && value.nsec.positive?)
```

Comment:
Checking `value.usec.positive?` is redundant, since if `value.usec.positive?` is true, `value.nsec.positive?` is also true.

Reply:
@kamipo I was trying to guard against the case we have a value that responds to `usec`, but *not* to `nsec`. If we don't need to handle that scenario, we certainly can just check `nsec.positive?` here.

Reply:
I think everything responds to `nsec` now (in Rails at least - DateTime doesn't without Active Support loaded, but it also doesn't respond to `usec` either 😄 )